roles/content_views/README.md

```diff
@@ -0,0 +1,64 @@
+theforeman.foreman.content_views
+=========================================
+
+This role creates and manages Content Views.
+
+Role Variables
+--------------
+
+This role supports the [Common Role Variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md#common-role-variables).
+
+The main data structure for this role is the list of `foreman_content_views`. Each Content View requires the following fields:
+
+- `name` - the name of the content view
+
+Each content view also requires either a list of repositories or components (for a composite content view):
+- `repositories` - list of repositories to add to the content view, each repository needs the following fields:
+  - `name` - the name of the repository
+  - `product` - the product the repository belongs to
+- `components` - list of content views to add to the composite content view, each component needs the following fields:
+  - `content_view` - the name of the content view
+  - `content_view_version` - the version of the content view to add, *or*
+  - `latest` - if `true`, the latest version of the content view will be used
```

Comment:
```suggestion
- `repositories` - List of repositories to add to the content view. Each repository requires the following fields:
  - `name` - The name of the repository
  - `product` - The product which the repository belongs to
- `components` - List of content views to add to the composite content view. Each component requires the following fields:
  - `content_view` - The name of the content view
  - `content_view_version` - The version of the content view to add, *or*
  - `latest` - If `true`, the latest version of the content view will be used
```

A few very small style nits here. "The product to which the repository belongs" would _technically_ be more formally correct English, but still sounds awkward even to a lot of native speakers. I settled on "The product which the repository belongs to" as a clear but less formal middle ground.